tests/test-poxl.clar

```diff
@@ -214,6 +213,11 @@
         (print "test-can-mine-tokens")
         (asserts! (is-eq (err ERR-STACKING-NOT-AVAILABLE) (can-mine-tokens 'SP1GYBXAJSEF8SY0ERKA068J93E3EGNTXHR98MM5P u0 u0 miners-rec)) (err u0))
         (asserts! (is-eq (err ERR-ROUND-FULL) (can-mine-tokens 'SP1GYBXAJSEF8SY0ERKA068J93E3EGNTXHR98MM5P u10 u0 full-miners-rec)) (err u1))
+        ;; arrange for next assert
+        (map-set miners-block-commitment 
+            { miner: 'SP1GYBXAJSEF8SY0ERKA068J93E3EGNTXHR98MM5P, stacks-block-height: u10 }
+            { committed: true }
+        )
```

Comment:
The way the tests work, this map is going to remain modified for the rest of the tests.  Not a bug, but I just want to make sure you're aware.